nms/app/fbcnms-projects/magmalte/app/views/equipment/EnodebDetailConfigEdit.js

```diff
@@ -81,10 +73,9 @@ const useStyles = makeStyles(_ => ({
       background: colors.primary.mirage,
     },
   },
-  input: {
-    display: 'inline-flex',
-    margin: '5px 0',
-    width: '80%',
+  tabBar: {
```

Comment:
Can't this also be abstracted out in mui themes? Do we have  variants for tabs within dialog and tabs outside ?

Reply:
So I looked for awhile through the Material UI docs to see if there was any type of theming that could be used to control the tabs background color, and I just could not find it. The best I could find was changing the font color, as well as the indicator slider below it.

I'm hesitant to make this a global change because I can see a world in which we may want tabs over top a light background, though from all the screens I've looked at in the design, I don't see any cases of that. If you think we're safe to pass this globally, I'm all for it... I really just was worried about affecting some other tabbed component.

Reply:
sure. thanks for looking into that. yes. that makes sense. for now we can just use this in all the associated components.